src/Controllers/Leaderboard.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * This file is part of CopyPatrol application
+ * Copyright (C) 2016  Niharika Kohli and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author MusikAnimal <musikanimal@wikimedia.org>
+ * @copyright © 2016 Niharika Kohli and contributors.
```

Comment:
At some point we need to change all these copyright notices to say "@copyright © 2016 Wikimedia Foundation", but that's not really relevant to this change.